seed/seed20_test.go

```diff
@@ -3520,6 +3520,7 @@ func (s *seed20Suite) TestCopy(c *C) {
 		"optional20-b_1.0.snap",
 		"local-component-test_1.0.snap",
 		"local-component-test+comp4_1.0.comp",
+		"component-test+comp3_44.comp",
```

Comment:
these are containers in the system local snaps dirs?

Reply:
Yes, I'll rename this field to something better.